Lib/asyncio/base_events.py

```diff
@@ -543,6 +550,16 @@ def _asyncgen_firstiter_hook(self, agen):
                     'asyncgen': agen
                 })
 
+    def shutdown_default_executor(self):
+        """Shutdown the default executor, but wait for the threads
```

Comment:
You need to fit the first sentence into one line.

Reply:
To avoid code duplication, you can add a `def _shutdown_default_executor(self, wait):` and call it from `loop.close()` and `loop. shutdown_default_executor`

Reply:
@1st1:
> You need to fit the first sentence into one line.

I think that would exceed the recommended 80 char limit per line, would you like me to fit the max words within the limit on each line or reduce the docstring sentence? I can make it a bit shorter:

> """Shutdown the default executor, but wait for the threadpool to finish."""

Reply:
Ah, good to know. Thanks.